packages/grafana-data/src/dataframe/DataFrameJSON.test.ts

```diff
@@ -83,5 +83,59 @@ describe('DataFrame JSON', () => {
         }
       `);
     });
+
+    it('should inflate values from enums', () => {
+      const json: DataFrameJSON = {
+        schema: {
+          fields: [
+            { name: 'time', type: FieldType.time },
+            { name: 'value', type: FieldType.number },
```

Comment:
🤔 somethign weird here, though maybe we just need more clear docs and examples.

Is "enum" only valid for string types?   What type should be defined where?  In typescript we can infer, in go I'm not sure that is a great pattern.

Still generally skeptical this is a real savings -- especially with gzip enabled.  But this will let us know :)